check-services.php

```diff
@@ -48,13 +48,9 @@
         if ($service_status != $status) {
             $update['service_changed'] = time();
         }
-        else {
-            unset($updated);
```

Comment:
I think that else was a relict anyways. figured the main issue here was the webui part that pulled the wrong var.